Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -98,14 +133,22 @@ private set
                     else
                     {
                         // Abort press.
-                        isPressing = false;
+                        IsPressing = false;
 
                         TouchEnd.Invoke();
                     }
                 }
             }
         }
 
+        /// <summary>
+        /// Represents the state of whether the button is currently being pressed.
+        /// </summary>
+        public bool IsPressing { get; private set; }
```

Comment:
Why are we exposing IsPressing but not IsTouching?

Reply:
we are exposing IsTouching . there's a public get for that just above the IsPressing